xbmc/cores/DllLoader/DllLoader.cpp

```diff
@@ -213,7 +213,7 @@ void DllLoader::PrintImportTable(ImportDirTable_t *ImportDirTable)
   ImportDirTable_t *Imp = ImportDirTable;
   int HavePrinted = 0;
 
-  CLog::Log(LOGDEBUG, "The Coff Image contains the following imports:\n\n");
+  CLog::Log(LOGDEBUG, "The Coff Image contains the following imports:\n");
```

Comment:
I always only removed the last `\n` because in some cases it looked like there's some kind of connection between multiple log messages.